db/migrate/20250214015507_create_ticket_comments.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateTicketComments < ActiveRecord::Migration[7.2]
+  def change
+    create_table :ticket_comments do |t|
+      t.references :ticket, null: false
```

Comment:
Why no foreign key here?

Reply:
(You do have foreign keys below for hte other two `references`, that's why I'm asking)

Reply:
I actually forgot it TBH, I've added it now.